probables/countminsketch/countminsketch.py

```diff
@@ -372,6 +373,19 @@ def __mean_min_query(self, results):
             res = meanmin[self.depth // 2]
         return res
 
+    @classmethod
```

Comment:
I would prefer if this was an object method instead of a class method (similar to the Bloom Filter union and intersection implementations). This would allow for the user to decide what object gets merged into and would reduce space for very large count-min sketches.

``` python
cms.merge(another_cms)
```

Reply:
If you'd prefer, I can take your code and make these changes. Sorry it took so long to get to this review as I have been busy

Reply:
Ok, feel free to take the code and make these changes.